how-to-play/index.md

```diff
@@ -28,8 +28,6 @@ When the ghost wants to try to kill you, it enters a hunting mode. Your flashlig
 
 After that, it will move through the house and if it is near you, could try and kill you!
 
-**During the hunt the radio on your shoulder does not work at all.**
```

Comment:
Why did you remove this?

Reply:
Didn't mean to May have gotten overwritten by the auto generated text that caused the pull error